base/deprecated.jl

```diff
@@ -1086,4 +1087,6 @@ eval(Base.LinAlg, quote
     end
 end)
 
+Filesystem.stop_watching(stream::Filesystem._FDWatcher) = depwarn("stop_watching(::_FDWatcher) is removed", :stop_watching)
```

Comment:
It's in deprecated.jl, you're calling depwarn - it's a deprecation, just with a no-op replacement other than the warning. If it were removed it wouldn't be here at all.